modules/decima-ext-model-exporter/src/main/java/com/shade/decima/ui/data/viewer/model/dmf/DMFTransform.java

```diff
@@ -1,34 +1,30 @@
 package com.shade.decima.ui.data.viewer.model.dmf;
 
-import com.shade.decima.ui.data.viewer.model.utils.Matrix4x4;
-import com.shade.decima.ui.data.viewer.model.utils.Quaternion;
-import com.shade.decima.ui.data.viewer.model.utils.Transform;
-import com.shade.decima.ui.data.viewer.model.utils.Vector3;
 import com.shade.util.NotNull;
+import org.joml.*;
 
 import java.util.Arrays;
 
 public class DMFTransform {
```

Comment:
So transform is just a matrix now. Can we replace it everywhere?

Reply:
Maintain backward compatability